src/lib/mdns/minimal/Server.cpp

```diff
@@ -291,22 +302,27 @@ CHIP_ERROR ServerBase::BroadcastSend(chip::System::PacketBufferHandle && data, u
 #endif
         else
         {
+            // This is a general error of internal consistency: every address has a known type
+            // Fail completly otherwise.
             return CHIP_ERROR_INCORRECT_STATE;
         }
 
-        if (err == chip::System::MapErrorPOSIX(ENETUNREACH))
+        if (err == CHIP_NO_ERROR)
         {
-            // Send attempted to an unreachable network. Generally should not happen if
-            // interfaces are configured properly, however such a failure to broadcast
-            // may not be critical either.
-            ChipLogError(Discovery, "Attempt to mDNS broadcast to an unreachable destination.");
+            hadSuccesfulSend = true;
```

Comment:
Maybe `ChipLogProgress` here, just so it's clear that something is succeeding?  Or do it at the end if `hadSuccessfulSend` is true after the loop?

Reply:
I believe errors are generally unexpected for the cases where minmdns cares about (embedded devices) hence the error log.

I can see how workstations would get this all the time (docker interfaces and such) however workstations are not the target for minmdns. I would change my mind if we show this is unnecesarely noisy for an embedded platform (maybe one with wifi and ethernet, where I could see how only some interface may work)

Reply:
I was just saying that we should also log when something succeeds, so if some succeed and some fail the log does not look like it all failed.

Reply:
I had misunderstood - thought you were asking to make 'error' into 'progress'.
I added the success log. I am concerned that it may be a bit noisy, however will see how it goes and if it is indeed nosiy we can trim it down again.